metrics-server/post_install.md

```diff
@@ -0,0 +1,31 @@
+## metrics-server - provide metrics for Kubernetes HPAv2 (Pod auto-scaling)
+
+### Test it out
+
+* Check that data is being received from nodes:
```

Comment:
Would rather drop these bullet points, normal paragraphs work well when interspersed with code snippets. If you like I can change these after merging though.

Reply:
Resolved